object_detection/setup.md

```diff
@@ -0,0 +1,59 @@
+## Setup Kubeflow
+### Requirements
+
+ - Kubernetes cluster
+ - Access to a working `kubectl` (Kubernetes CLI)
+ - Ksonnet CLI: [ks](https://ksonnet.io/)
+
+### Setup
+Refer to the [user guide](https://github.com/kubeflow/kubeflow/blob/master/user_guide.md) for instructions on how to setup kubeflow on your kubernetes cluster. Specifically, look at the section on [deploying kubeflow](https://github.com/kubeflow/kubeflow/blob/master/user_guide.md#deploy-kubeflow).
```

Comment:
Update link to https://www.kubeflow.org/docs/about/user_guide/

Reply:
Good catch! thanks @ankushagarwal!, I have updated the links to the new location of the user guide